### ISSUE TITLE
MVM datasets using filters on the bluer end of the spectrum have trouble aligning to GAIA

### ISSUE THREAD

**Issue by stscijgbot-hstdp** [spacetelescope/drizzlepac#1030]
_Issue [HLA-554](https://jira.stsci.edu/browse/HLA-554) was created on JIRA by [Michael Dulude](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dulude):_

MVM observations using filters on the bluer side of the spectrum have trouble aligning to GAIA (which is focused more on the redder end of the spectrum) simply because many of the GAIA sources are very dim or undetectable at bluer bandpasses. 
Example: 
jc6e03 (ACS/SBC observation 13158_03)
Filter: F122M
There are 347 GAIA sources in the field, but none of them really line up to any of the sources in the field. 

The goal of this ticket is to identify where in the process the alignment fails. Based on that, we can follow up with other tickets to address the underlying issue.

**Comment by stscijgbot-hstdp** [comment]
_Comment by [Warren Hack](https://jira.stsci.edu/secure/ViewProfile.jspa?name=hack) on [JIRA](https://jira.stsci.edu/browse/HLA-554?focusedCommentId=563352#comment-563352):_

The data for visit 'jc6e03' includes ACS/SBC F122M images along with ACS/SBC Prism images.  As a result, the final drizzle product gets made using the a priori WCS which results in the GAIA sources not being aligned well to the field (as shown in the screen shot).  However, the F122M images were successfully aligned to GAIAeDR3, and when that WCS is used, it shows that at least some GAIA sources do align reasonably well in the field.  This can be seen in the display:   !GAIA_vs_FLT_GAIAeDR3wcs.png|thumbnail!

In short, there are instances where the far-UV images taken by ACS/HRC and ACS/SBC make it difficult to align to GAIA, but it works well for this visit.

**Comment by stsci-hack** [comment]
The remaining issues with these datasets comes down to passband differences between the exposures and the GAIA catalog.  Specifically, GAIA sources may not even be detectable in the HST images as GAIA measured them in visible/red bands which favors red sources with little UV emission.  Thus, the algorithms used do the best job possible to recognize identifiable sources to try to match to GAIA.

**Comment by stscijgbot-hstdp** [comment]
_Comment by [Warren Hack](https://jira.stsci.edu/secure/ViewProfile.jspa?name=hack) on [JIRA](https://jira.stsci.edu/browse/HLA-554?focusedCommentId=563352#comment-563352):_

The data for visit 'jc6e03' includes ACS/SBC F122M images along with ACS/SBC Prism images.  As a result, the final drizzle product gets made using the a priori WCS which results in the GAIA sources not being aligned well to the field (as shown in the screen shot).  However, the F122M images were successfully aligned to GAIAeDR3, and when that WCS is used, it shows that at least some GAIA sources do align reasonably well in the field.  This can be seen in the display:   !GAIA_vs_FLT_GAIAeDR3wcs.png|thumbnail!

In short, there are instances where the far-UV images taken by ACS/HRC and ACS/SBC make it difficult to align to GAIA, but it works well for this visit.

**Comment by stscijgbot-hstdp** [comment]
_Comment by [Michele De La Pena](https://jira.stsci.edu/secure/ViewProfile.jspa?name=mdelapena) on [JIRA](https://jira.stsci.edu/browse/HLA-554?focusedCommentId=644731#comment-644731):_

The corresponding Git PR#&#x2060;1030 was closed on 12 January 2022.

**Comment by stscijgbot-hstdp** [comment]
_Comment by [Michele De La Pena](https://jira.stsci.edu/secure/ViewProfile.jspa?name=mdelapena) on [JIRA](https://jira.stsci.edu/browse/HLA-554?focusedCommentId=644731#comment-644731):_

The corresponding Git PR#&#x2060;1030 was closed on 12 January 2022.
